### PR TITLE
add image link, edit readme file(Highlites)

### PR DESCRIPTION
**Pull request by YoonSuhyeok**
firefox-devtools/ux#120

https://github.com/firefox-devtools/firefox-devtools.github.io/pull/41
⬆ I checked some comments.  Follow the comments, I fixed readme.file!
🎉Thank you! violasong!!🎉

violasong want to add the image. -> I add image from https://www.mozilla.org/en-US/firefox/developer/
image
image

I fixed it. I hope you want it. 🙏

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
...
 # About DevTools
 
+## Highlights
+
+* [Inactive CSS](https://hacks.mozilla.org/2019/10/firefox-70-a-bountiful-release-for-all/#developertools):
+ Firefox DevTools now grays out CSS declarations that don’t have an effect on the page. When you hover over the info icon, you’ll see a useful message about why the CSS is not being applied, including a hint about how to fix the problem.
+ ![Inactive_CSS](https://www.mozilla.org/media/img/firefox/developer/hero-inactive-css.40a2e0b8ddb1.png) 
+
+* [Master CSS](https://mozilladevelopers.github.io/playground/css-grid/):
+ Grid Firefox is the only browser with tools built specifically for building and designing with CSS Grid. These tools allow you to visualize the grid, display associated area names, preview transformations on the grid and much more.
```
Comment (violasong) [inline README.md:43]: The word "Grid" should be inside the link :)

File: README.md [hunk 2]
```diff
@@ -33,6 +33,24 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
 
 # About DevTools
 
+## Highlights
+
+* [Inactive CSS](https://hacks.mozilla.org/2019/10/firefox-70-a-bountiful-release-for-all/#developertools):
+ Firefox DevTools now grays out CSS declarations that don’t have an effect on the page. When you hover over the info icon, you’ll see a useful message about why the CSS is not being applied, including a hint about how to fix the problem.
+ ![Inactive_CSS](https://www.mozilla.org/media/img/firefox/developer/hero-inactive-css.40a2e0b8ddb1.png)
```
Comment (violasong) [inline README.md:40]: Thanks for trying out these images! I think some more graphic design work needs to happen for them to work with this page, however. Let's leave them out of this patch for now.

File: README.md (hunk 1 above)
Comment (YoonSuhyeok) [reply README.md:43]: I got it!

File: README.md (hunk 2 above)
Comment (YoonSuhyeok) [reply README.md:40]: Okay!

**Pull request closed without merging**